log-collector-script/windows/eks-log-collector.ps1

```diff
@@ -349,6 +350,25 @@ Function get_network_info{
     }
 }
 
+Function get_windows_events{
+    try {
+        Write-Host "Collecting Windows events"
+        Copy-Item "$env:SystemDrive\Windows\System32\Winevt\Logs\Application.evtx" -Destination $info_system\events
```

Comment:
Do you intend to keep them as event files? The event format is easy to view in Windows instances, but will be difficult to view in other OS machines.

Reply:
Yes you are right this will be difficult to view on Mac but there are 3party tools which can be used to view on mac. They can be exported to csv from event file if that turns out to be the case. 
There is benefit to viewing in Windows though, It will be easy to sort events in Windows.